docs-web/installation/installation-with-helm.md

```diff
@@ -37,73 +37,40 @@ $ helm repo update
-
-  * Using Helm 2.x client:
-
-      For NGINX:
-      ```console
-      $ helm install --name my-release nginx-edge/nginx-ingress --devel
-      ```
-
-      For NGINX Plus: (assuming you have pushed the Ingress controller image `nginx-plus-ingress` to your private registry `myregistry.example.com`)
-      ```console
-      $ helm install --name my-release nginx-edge/nginx-ingress --set controller.image.repository=myregistry.example.com/nginx-plus-ingress --set controller.nginxplus=true --devel
-      ```
+For NGINX:
+```console
+$ helm install my-release nginx-edge/nginx-ingress --devel
+```
 
+For NGINX Plus: (assuming you have pushed the Ingress controller image `nginx-plus-ingress` to your private registry `myregistry.example.com`)
+```console
+$ helm install my-release nginx-edge/nginx-ingress --set controller.image.repository=myregistry.example.com/nginx-plus-ingress --set controller.nginxplus=true --devel
+```
 ### Installing Using Chart Sources
 
 To install the chart with the release name my-release (my-release is the name that you choose):
 
-  * Using Helm 3.x client:
-
-      For NGINX:
-      ```console
-      $ helm install my-release .
-      ```
-
-      For NGINX Plus:
-      ```console
-      $ helm install my-release -f values-plus.yaml .
-      ```
-
-  * Using Helm 2.x client:
-
-      For NGINX:
-      ```console
-      $ helm install --name my-release .
-      ```
+For NGINX:
+```console
+$ helm install my-release .
+```
 
-      For NGINX Plus:
-      ```console
-      $ helm install --name my-release -f values-plus.yaml .
-      ```
+For NGINX Plus:
+```console
+$ helm install my-release -f values-plus.yaml .
+```
 
-    The command deploys the Ingress controller in your Kubernetes cluster in the default configuration. The configuration section lists the parameters that can be configured during installation.
+The command deploys the Ingress controller in your Kubernetes cluster in the default configuration. The configuration section lists the parameters that can be configured during installation.
 
-    When deploying the Ingress controller, make sure to use your own TLS certificate and key for the default server rather than the default pre-generated ones. Read the [Configuration](#configuration) section below to see how to configure a TLS certificate and key for the default server. Note that the default server returns the Not Found page with the 404 status code for all requests for domains for which there are no Ingress rules defined.
+When deploying the Ingress controller, make sure to use your own TLS certificate and key for the default server rather than the default pre-generated ones. Read the [Configuration](#configuration) section below to see how to configure a TLS certificate and key for the default server. Note that the default server returns the Not Found page with the 404 status code for all requests for domains for which there are no Ingress rules defined.
 
 > **Tip**: List all releases using `helm list`
```

Comment:
Personally, I think we can remove this:
```suggestion
```

I prefer to focus on documenting the details of our chart implementation, I think it's fair to assume the reader knows how to use the basics of helm cli. - I don't feel strongly about this though.